src/management-system-v2/components/processes.tsx

```diff
@@ -131,14 +135,28 @@ const Processes: FC = () => {
           }}
         />
       </Tooltip>
-      {ability.can('delete', 'Process') && (
+
+      <AuthCan action="delete" resource={toCaslResource('Process', process)}>
```

Comment:
Can you use ConfirmationButton instead of PopConfirm

Reply:
Replaced PopConfirm with ConfirmationButton